src/services/e2e.ts

```diff
@@ -21,6 +21,9 @@ const $ = {
   mockServer: token('mockServer'),
   mock: token<Mocker>('mocker'),
   Env: token('Env'),
+  client: token<{
+    history: HistoryEntry[]
+  }>('client'),
```

Comment:
This will become a more formally defined 'client' in a soon to come PR, I want to investigate a few things before I do that tho.

Reply:
I've now made a more formal `client`